share/locale/en.json

```diff
@@ -95,6 +95,10 @@
     "gui_settings_tor_bridges_no_bridges_radio_option": "Don't use bridges",
     "gui_settings_tor_bridges_obfs4_radio_option": "Use built-in obfs4 pluggable transports",
     "gui_settings_tor_bridges_obfs4_radio_option_no_obfs4proxy": "Use built-in obfs4 pluggable transports (requires obfs4proxy)",
+    "gui_settings_tor_bridges_meek_lite_amazon_radio_option": "Use built-in meek_lite_amazon pluggable transports",
+    "gui_settings_tor_bridges_meek_lite_amazon_radio_option_no_meek_lite_obfs4proxy": "Use built-in meek_lite_amazon pluggable transports (requires obfs4proxy)",
+    "gui_settings_tor_bridges_meek_lite_azure_radio_option": "Use built-in meek_lite_azure pluggable transports",
+    "gui_settings_tor_bridges_meek_lite_azure_radio_option_no_obfs4proxy": "Use built-in meek_lite_azure pluggable transports (requires obfs4proxy)",
```

Comment:
The extra underscores in the bridge names look a bit awkward in the Settings UI. Can I suggest to make them a bit more 'friendly' to read, like this:

Use built-in meek_lite (Amazon) pluggable transports
Use built-in meek_lite (Amazon) pluggable transports (requires obfs4proxy)

Use built-in meek_lite (Azure) pluggable transports
Use built-in meek_lite (Azure) pluggable transports (requires obfs4proxy)

I'm ok with the underscore in 'meek_lite' since that's the official name.